test/sql/test_select.cpp

```diff
@@ -40,86 +39,84 @@ BOOST_AUTO_TEST_CASE(test_data_type) {
   std::string sql = "select name from student where ";
   std::string cond = "birthday > '2001-12-06'";
   std::string semi = ";";
-  auto sw_cond = [&sql, &semi](const std::string& cond) -> std::string {
-    return sql + cond + semi;
-  };
+  auto sw_cond = [&sql, &semi](const std::string &cond) -> std::string { return sql + cond + semi; };
 
   ParserContext pc;
```

Comment:
previous lexer for string literal has a fatal error, though I'm not sure why it occurs somehow, I found [it](https://stackoverflow.com/questions/2039795/regular-expression-for-a-string-literal-in-flex-lex) useful to solve this.